recipe/meta.yaml

```diff
@@ -44,33 +44,36 @@ outputs:
         - distributed
         - ecmwf-api-client
         - eofs
-        - esmpy
-        - esmvalcore 2.10.*
+        - esmpy !=8.1.0,<8.6.0  # https://github.com/SciTools-incubator/iris-esmf-regrid/pull/342#issuecomment-2092921514
+        - esmvalcore 2.11.*
         - fiona
         - fire
+        - fsspec
         - gdal
+        - importlib_metadata <8  # required for Python < 3.10
         - iris >=3.6.1
         - iris-esmf-regrid >=0.7.0
         - jinja2
         - joblib
         - lime
         - mapgenerator >=1.0.5
-        - matplotlib-base
+        - matplotlib-base <3.9.0
         - natsort
         - nc-time-axis
         - netCDF4
         - numba
         - numpy !=1.24.3  # severe masking bug
```

Comment:
I think it's best we pinned this one too to `<2.0.0` for now - the pin is active and will be enforced from `esmvalcore` but it may be that we will have that pin released before Tool, and we'd still want to keep it to <2 in Tool, until we iron out all the bugs - what say you @ehogan and @bouweandela

Reply:
It wasn't pinned in the [environment.yml](https://github.com/ESMValGroup/ESMValTool/blob/v2.11.x/environment.yml#L42) file, so I assumed it was intentional? Happy to go with what you and @bouweandela agree 👍

Reply:
I agree it would be the safest option to pin numpy < 2

Reply:
Done in 449e587 😊